stdlib/nodejs/kernel.rb

```diff
@@ -16,6 +16,7 @@ def caller(*args)
     }
   end
 
+  # @deprecated Please use `require('module')` instead
   def node_require(path)
```

Comment:
Can you add a `warn` inside the method too? 
Shouldn't bother users too much since it happens only once when files are initially required?

Reply:
I added the `warn` message using the GitHub UI 😨

Reply:
@elia Does the deprecation message look good ?

Reply:
@Mogztter it looks good, only a nitpick, you should remove the backticks around `node_require` or make clear `require('module')` needs to be wrapped in them.

Reply:
Indeed, I will remove the backticks around `node_require`, my bad